web_infrastructure/auth0_user.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/python
+#
+# (c) 2015, RSD Services S.A
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: auth0_user
+short_description: Module for managing auth0 users
+description:
+    - Auth0 user managing module. It can search for a given user and
+      return it's payload, create/delete user and update application
+      metadata.
+requirements: [ python requests ]
+version_added: "2.2"
+options:
+  state:
+    description:
+     - the state of the collection. Either present or absent
```

Comment:
1) You don't need the `Either present or absent`, the `choices` section will be automatically added to the documentation

2) Is it worth defining how 'delete_all' is different to 'absent'